<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="PENNANT-VSP112" failures="15" tests="53" name="Default test" time="1.304" errors="0" timestamp="2023-07-04T18:30:38 IST">
  <testcase classname="com.avengers.bus.services.test.FetchCountImplTest" name="testGetBusCount" time="0.023"/>
  <testcase classname="com.avengers.bus.services.test.FetchCountImplTest" name="testGetCollection" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.FetchCountImplTest" name="testGetFutureTicketCollection" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.FetchCountImplTest" name="testGetPastTicketCollection" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.FetchCountImplTest" name="testGetRouteCount" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.FetchCountImplTest" name="testGetServiceCount" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.FetchCountImplTest" name="testGetTripCount" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.FetchCountImplTest" name="testGetUserCount" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.FetchUserCountsImplTest" name="testGetUserBookedCount" time="0.018">
    <failure type="org.mockito.exceptions.verification.WantedButNotInvoked" message="
Wanted but not invoked:
logger.info(
    &amp;quot;Fetching User Booked Count for user with ID: {}&amp;quot;,
    1
);
-&amp;gt; at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserBookedCount(FetchUserCountsImplTest.java:43)
Actually, there were zero interactions with this mock.
">
      <![CDATA[Wanted but not invoked:
logger.info(
    "Fetching User Booked Count for user with ID: {}",
    1
);
-> at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserBookedCount(FetchUserCountsImplTest.java:43)
Actually, there were zero interactions with this mock.

at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserBookedCount(FetchUserCountsImplTest.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testGetUserBookedCount -->
  <testcase classname="com.avengers.bus.services.test.FetchUserCountsImplTest" name="testGetUserCounts" time="0.002">
    <failure type="org.mockito.exceptions.verification.WantedButNotInvoked" message="
Wanted but not invoked:
logger.info(
    &amp;quot;Fetching User Counts for user with ID: {}&amp;quot;,
    1
);
-&amp;gt; at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserCounts(FetchUserCountsImplTest.java:111)
Actually, there were zero interactions with this mock.
">
      <![CDATA[Wanted but not invoked:
logger.info(
    "Fetching User Counts for user with ID: {}",
    1
);
-> at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserCounts(FetchUserCountsImplTest.java:111)
Actually, there were zero interactions with this mock.

at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserCounts(FetchUserCountsImplTest.java:111)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testGetUserCounts -->
  <testcase classname="com.avengers.bus.services.test.FetchUserCountsImplTest" name="testGetUserFutureTravelCount" time="0.001">
    <failure type="org.mockito.exceptions.verification.WantedButNotInvoked" message="
Wanted but not invoked:
logger.info(
    &amp;quot;Fetching User Future Travel Count for user with ID: {}&amp;quot;,
    1
);
-&amp;gt; at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserFutureTravelCount(FetchUserCountsImplTest.java:134)
Actually, there were zero interactions with this mock.
">
      <![CDATA[Wanted but not invoked:
logger.info(
    "Fetching User Future Travel Count for user with ID: {}",
    1
);
-> at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserFutureTravelCount(FetchUserCountsImplTest.java:134)
Actually, there were zero interactions with this mock.

at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserFutureTravelCount(FetchUserCountsImplTest.java:134)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testGetUserFutureTravelCount -->
  <testcase classname="com.avengers.bus.services.test.FetchUserCountsImplTest" name="testGetUserTravelCount" time="0.001">
    <failure type="org.mockito.exceptions.verification.WantedButNotInvoked" message="
Wanted but not invoked:
logger.info(
    &amp;quot;Fetching User Travel Count for user with ID: {}&amp;quot;,
    1
);
-&amp;gt; at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserTravelCount(FetchUserCountsImplTest.java:58)
Actually, there were zero interactions with this mock.
">
      <![CDATA[Wanted but not invoked:
logger.info(
    "Fetching User Travel Count for user with ID: {}",
    1
);
-> at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserTravelCount(FetchUserCountsImplTest.java:58)
Actually, there were zero interactions with this mock.

at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserTravelCount(FetchUserCountsImplTest.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testGetUserTravelCount -->
  <testcase classname="com.avengers.bus.services.test.FetchUserCountsImplTest" name="testGetUserWallet" time="0.002">
    <failure type="org.mockito.exceptions.verification.WantedButNotInvoked" message="
Wanted but not invoked:
logger.info(
    &amp;quot;Fetching User Wallet for user with ID: {}&amp;quot;,
    1
);
-&amp;gt; at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserWallet(FetchUserCountsImplTest.java:73)
Actually, there were zero interactions with this mock.
">
      <![CDATA[Wanted but not invoked:
logger.info(
    "Fetching User Wallet for user with ID: {}",
    1
);
-> at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserWallet(FetchUserCountsImplTest.java:73)
Actually, there were zero interactions with this mock.

at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUserWallet(FetchUserCountsImplTest.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testGetUserWallet -->
  <testcase classname="com.avengers.bus.services.test.FetchUserCountsImplTest" name="testGetUsername" time="0.003">
    <failure type="org.mockito.exceptions.verification.WantedButNotInvoked" message="
Wanted but not invoked:
logger.info(
    &amp;quot;Fetching Username for user with ID: {}&amp;quot;,
    1
);
-&amp;gt; at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUsername(FetchUserCountsImplTest.java:88)
Actually, there were zero interactions with this mock.
">
      <![CDATA[Wanted but not invoked:
logger.info(
    "Fetching Username for user with ID: {}",
    1
);
-> at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUsername(FetchUserCountsImplTest.java:88)
Actually, there were zero interactions with this mock.

at com.avengers.bus.services.test.FetchUserCountsImplTest.testGetUsername(FetchUserCountsImplTest.java:88)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testGetUsername -->
  <testcase classname="com.avengers.bus.services.test.TicketDetailsImplTest" name="testGenerateTicketDetails" time="0.003">
    <failure type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException
at java.sql/java.sql.Date.valueOf(Date.java:141)
at com.avengers.bus.models.dtoModels.BusSearchListDto.setTrip_date(BusSearchListDto.java:85)
at com.avengers.bus.services.test.TicketDetailsImplTest.testGenerateTicketDetails(TicketDetailsImplTest.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testGenerateTicketDetails -->
  <testcase classname="com.avengers.bus.services.test.CancelTicketServiceImplTest" name="cancelTicket_InvalidBookingId_ReturnsFalse" time="0.003"/>
  <testcase classname="com.avengers.bus.services.test.CancelTicketServiceImplTest" name="cancelTicket_NullBookingId_ReturnsFalse" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.CancelTicketServiceImplTest" name="testCancelTicket" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.BusListServiceImplTest" name="testGetBusListWithTravelClass" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.BusListServiceImplTest" name="testGetBusListWithoutTravelClass" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.FetchUserImplTest" name="testAddUser" time="0.002">
    <failure type="org.mockito.exceptions.verification.WantedButNotInvoked" message="
Wanted but not invoked:
logger.info(
    &amp;quot;Adding or updating user with ID: {}&amp;quot;,
    1
);
-&amp;gt; at com.avengers.bus.services.test.FetchUserImplTest.testAddUser(FetchUserImplTest.java:91)
Actually, there were zero interactions with this mock.
">
      <![CDATA[Wanted but not invoked:
logger.info(
    "Adding or updating user with ID: {}",
    1
);
-> at com.avengers.bus.services.test.FetchUserImplTest.testAddUser(FetchUserImplTest.java:91)
Actually, there were zero interactions with this mock.

at com.avengers.bus.services.test.FetchUserImplTest.testAddUser(FetchUserImplTest.java:91)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testAddUser -->
  <testcase classname="com.avengers.bus.services.test.FetchUserImplTest" name="testGetUserList" time="0.289">
    <failure type="org.mockito.exceptions.verification.WantedButNotInvoked" message="
Wanted but not invoked:
logger.info(
    &amp;quot;Fetching user details for user with ID: {}&amp;quot;,
    1
);
-&amp;gt; at com.avengers.bus.services.test.FetchUserImplTest.testGetUserList(FetchUserImplTest.java:61)
Actually, there were zero interactions with this mock.
">
      <![CDATA[Wanted but not invoked:
logger.info(
    "Fetching user details for user with ID: {}",
    1
);
-> at com.avengers.bus.services.test.FetchUserImplTest.testGetUserList(FetchUserImplTest.java:61)
Actually, there were zero interactions with this mock.

at com.avengers.bus.services.test.FetchUserImplTest.testGetUserList(FetchUserImplTest.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testGetUserList -->
  <testcase classname="com.avengers.bus.services.test.FetchUserImplTest" name="testGetUserListException" time="0.001">
    <failure type="org.mockito.exceptions.verification.WantedButNotInvoked" message="
Wanted but not invoked:
logger.info(
    &amp;quot;Fetching user details for user with ID: {}&amp;quot;,
    1
);
-&amp;gt; at com.avengers.bus.services.test.FetchUserImplTest.testGetUserListException(FetchUserImplTest.java:75)
Actually, there were zero interactions with this mock.
">
      <![CDATA[Wanted but not invoked:
logger.info(
    "Fetching user details for user with ID: {}",
    1
);
-> at com.avengers.bus.services.test.FetchUserImplTest.testGetUserListException(FetchUserImplTest.java:75)
Actually, there were zero interactions with this mock.

at com.avengers.bus.services.test.FetchUserImplTest.testGetUserListException(FetchUserImplTest.java:75)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testGetUserListException -->
  <testcase classname="com.avengers.bus.services.test.FetchUserTicketsImplTest" name="testGetPastTicketList" time="0.003">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.util.List.iterator()&amp;quot; because the return value of &amp;quot;com.avengers.bus.models.entityModels.Ticket.getPassengerList()&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.util.List.iterator()" because the return value of "com.avengers.bus.models.entityModels.Ticket.getPassengerList()" is null
at com.avengers.bus.services.implementation.FetchUserTicketsImpl.getPastTicketList(FetchUserTicketsImpl.java:73)
at com.avengers.bus.services.test.FetchUserTicketsImplTest.testGetPastTicketList(FetchUserTicketsImplTest.java:81)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testGetPastTicketList -->
  <testcase classname="com.avengers.bus.services.test.FetchUserTicketsImplTest" name="testGetTicketList" time="0.001">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.util.List.iterator()&amp;quot; because the return value of &amp;quot;com.avengers.bus.models.entityModels.Ticket.getPassengerList()&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.util.List.iterator()" because the return value of "com.avengers.bus.models.entityModels.Ticket.getPassengerList()" is null
at com.avengers.bus.services.implementation.FetchUserTicketsImpl.getticketList(FetchUserTicketsImpl.java:37)
at com.avengers.bus.services.test.FetchUserTicketsImplTest.testGetTicketList(FetchUserTicketsImplTest.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testGetTicketList -->
  <testcase classname="com.avengers.bus.services.test.FetchUserPassengersImplTest" name="testAddUserPassengers" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.FetchUserPassengersImplTest" name="testAddeditedPassengers" time="0.005">
    <failure type="org.mockito.exceptions.verification.ArgumentsAreDifferent" message="
Argument(s) are different! Wanted:
userPassengersDAO.updateUserPassengers(
    UserPassengersId [user_id=0, passenger_name=null, age=0, gender=null]
);
-&amp;gt; at com.avengers.bus.services.test.FetchUserPassengersImplTest.testAddeditedPassengers(FetchUserPassengersImplTest.java:65)
Actual invocation has different arguments:
userPassengersDAO.updateUserPassengers(
    UserPassengersId [user_id=0, passenger_name=null, age=0, gender=null]
);
-&amp;gt; at com.avengers.bus.services.implementation.FetchUserPassengersImpl.addeditedpassengers(FetchUserPassengersImpl.java:56)
">
      <![CDATA[Argument(s) are different! Wanted:
userPassengersDAO.updateUserPassengers(
    UserPassengersId [user_id=0, passenger_name=null, age=0, gender=null]
);
-> at com.avengers.bus.services.test.FetchUserPassengersImplTest.testAddeditedPassengers(FetchUserPassengersImplTest.java:65)
Actual invocation has different arguments:
userPassengersDAO.updateUserPassengers(
    UserPassengersId [user_id=0, passenger_name=null, age=0, gender=null]
);
-> at com.avengers.bus.services.implementation.FetchUserPassengersImpl.addeditedpassengers(FetchUserPassengersImpl.java:56)

at com.avengers.bus.services.test.FetchUserPassengersImplTest.testAddeditedPassengers(FetchUserPassengersImplTest.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testAddeditedPassengers -->
  <testcase classname="com.avengers.bus.services.test.FetchUserPassengersImplTest" name="testDeleteUserPassengers" time="0.002">
    <failure type="org.mockito.exceptions.verification.ArgumentsAreDifferent" message="
Argument(s) are different! Wanted:
userPassengersDAO.deleteUserPassengers(
    UserPassengersId [user_id=0, passenger_name=null, age=0, gender=null]
);
-&amp;gt; at com.avengers.bus.services.test.FetchUserPassengersImplTest.testDeleteUserPassengers(FetchUserPassengersImplTest.java:76)
Actual invocation has different arguments:
userPassengersDAO.deleteUserPassengers(
    UserPassengersId [user_id=0, passenger_name=null, age=0, gender=null]
);
-&amp;gt; at com.avengers.bus.services.implementation.FetchUserPassengersImpl.deleteUserpassengers(FetchUserPassengersImpl.java:72)
">
      <![CDATA[Argument(s) are different! Wanted:
userPassengersDAO.deleteUserPassengers(
    UserPassengersId [user_id=0, passenger_name=null, age=0, gender=null]
);
-> at com.avengers.bus.services.test.FetchUserPassengersImplTest.testDeleteUserPassengers(FetchUserPassengersImplTest.java:76)
Actual invocation has different arguments:
userPassengersDAO.deleteUserPassengers(
    UserPassengersId [user_id=0, passenger_name=null, age=0, gender=null]
);
-> at com.avengers.bus.services.implementation.FetchUserPassengersImpl.deleteUserpassengers(FetchUserPassengersImpl.java:72)

at com.avengers.bus.services.test.FetchUserPassengersImplTest.testDeleteUserPassengers(FetchUserPassengersImplTest.java:76)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testDeleteUserPassengers -->
  <testcase classname="com.avengers.bus.services.test.FetchUserPassengersImplTest" name="testGetPassengersList" time="0.005"/>
  <testcase classname="com.avengers.bus.services.test.StationServiceImplTest" name="testListAll" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.StationServiceImplTest" name="testListAllWithEmptyStations" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.StationServiceImplTest" name="testListAllWithException" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.StationServiceImplTest" name="testListAllWithLargeNumberOfStations" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.StationServiceImplTest" name="testListAllWithMultipleStations" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.StationServiceImplTest" name="testListAllWithNoStations" time="0.0"/>
  <testcase classname="com.avengers.bus.services.test.AdminCountsServiceImplTest" name="getCountsTest" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.FetchDetailsImplTest" name="testGetRouteDetails" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.FetchDetailsImplTest" name="testGetServiceDetails" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.FetchGraphDataImplTest" name="testGetMonthlyCollection" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.FetchGraphDataImplTest" name="testGetRouteCollection" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.FetchGraphDataImplTest" name="testGetTripCollection" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.CheckSeatImplTest" name="checkAvailability_SeatsAvailable_ReturnsTrue" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.CheckSeatImplTest" name="checkAvailability_SeatsNotAvailable_ReturnsFalse" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.SaveTicketServiceImplTest" name="testSaveTheTicket" time="0.006">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:222)
at java.base/java.util.ImmutableCollections$AbstractImmutableList.indexOf(ImmutableCollections.java:166)
at com.avengers.bus.services.implementation.SaveTicketServiceImpl$1.compare(SaveTicketServiceImpl.java:66)
at com.avengers.bus.services.implementation.SaveTicketServiceImpl$1.compare(SaveTicketServiceImpl.java:1)
at java.base/java.util.TimSort.countRunAndMakeAscending(TimSort.java:355)
at java.base/java.util.TimSort.sort(TimSort.java:220)
at java.base/java.util.Arrays.sort(Arrays.java:1306)
at java.base/java.util.ArrayList.sort(ArrayList.java:1720)
at java.base/java.util.Collections.sort(Collections.java:179)
at com.avengers.bus.services.implementation.SaveTicketServiceImpl.saveTheTicket(SaveTicketServiceImpl.java:72)
at com.avengers.bus.services.test.SaveTicketServiceImplTest.testSaveTheTicket(SaveTicketServiceImplTest.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testSaveTheTicket -->
  <testcase classname="com.avengers.bus.services.test.UserPassengersServiceImplTest" name="testGetAllUserPassengers_InvalidUserId_ReturnsEmptyList" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.UserPassengersServiceImplTest" name="testGetAllUserPassengers_MultipleUsers_ReturnsCorrectPassengersForUserId" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.UserPassengersServiceImplTest" name="testGetAllUserPassengers_UserIdWithNoPassengers_ReturnsEmptyList" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.UserPassengersServiceImplTest" name="testGetAllUserPassengers_ValidUserId_ReturnsCorrectNumberOfPassengers" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.UserPassengersServiceImplTest" name="testGetAllUserPassengers_ValidUserId_ReturnsListOfUserPassengers" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.BusTypeServiceImplTest" name="testListAll" time="0.002"/>
  <testcase classname="com.avengers.bus.services.test.BusTypeServiceImplTest" name="testListAll_NoBusTypes" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.BusTypeServiceImplTest" name="testListAll_NullBusTypes" time="0.001"/>
  <testcase classname="com.avengers.bus.services.test.BusTypeServiceImplTest" name="testListAll_ReturnsMultipleBusTypes" time="0.001"/>
</testsuite> <!-- Default test -->
